datumaro/plugins/datumaro_format/converter.py

```diff
@@ -383,3 +386,41 @@ def patch(cls, dataset, patch, save_dir, **kwargs):
             )
             if osp.isdir(related_images_path):
                 shutil.rmtree(related_images_path)
+
+    @staticmethod
+    def _filterout_for_single_selection(item, categories):
+        if AnnotationType.label not in categories:
+            return item
+
+        name2parent_ss = {}
+        for label_category in categories[AnnotationType.label]:
+            name2parent_ss[label_category.name] = (
+                label_category.parent,
+                label_category.single_selection,
+            )
```

Comment:
Do we need to initialize this lookup dictionary for every item again and again? We can escalate this to the upper level and pass `name2parent_ss` to this function.

Reply:
Indeed, I change that making name2parent_ss once per each export.